modules/standard/Curl.chpl

```diff
@@ -90,181 +430,355 @@ proc slist.append(str:string) {
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_USERPWD.html */
 extern const curlopt_userpwd                    : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_PROXYUSERPWD.html */
 extern const curlopt_proxyuserpwd               : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_RANGE.html */
 extern const curlopt_range                      : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_READDATA.html */
 extern const curlopt_infile                     : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_ERRORBUFFER.html */
 extern const curlopt_errorbuffer                : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_WRITEFUNCTION.html */
 extern const curlopt_writefunction              : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_READFUNCTION.html */
 extern const curlopt_readfunction               : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_TIMEOUT.html */
 extern const curlopt_timeout                    : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_INFILESIZE.html */
 extern const curlopt_infilesize                 : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_POSTFIELDS.html */
 extern const curlopt_postfields                 : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_REFERER.html */
 extern const curlopt_referer                    : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_FTPPORT.html */
 extern const curlopt_ftpport                    : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_USERAGENT.html */
 extern const curlopt_useragent                  : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_LOW_SPEED_LIMIT.html */
 extern const curlopt_low_speed_limit            : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_LOW_SPEED_TIME.html */
 extern const curlopt_low_speed_time             : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_RESUME_FROM.html */
 extern const curlopt_resume_from                : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_COOKIE.html */
 extern const curlopt_cookie                     : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_HTTPHEADER.html */
 extern const curlopt_httpheader                 : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_HTTPPOST.html */
 extern const curlopt_httppost                   : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_SSLCERT.html */
 extern const curlopt_sslcert                    : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_KEYPASSWD.html */
 extern const curlopt_keypasswd                  : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_CRLF.html */
 extern const curlopt_crlf                       : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_QUOTE.html */
 extern const curlopt_quote                      : c_int ;
+/* See http://curl.haxx.se/libcurl/c/CURLOPT_HEADERDATA.html */
 extern const curlopt_writeheader                : c_int ;
```

Comment:
Pointing out that writeheader does this as well, feel free to ignore if you don't want to point the difference out :)